actors/miner/src/lib.rs

```diff
@@ -2129,12 +2113,21 @@ impl Actor {
                     "failed to load pre-committed sectors",
                 )
             })?;
-        confirm_sector_proofs_valid_internal(
+
+        let data_activations: Vec<DealsActivationInput> =
+            precommited_sectors.iter().map(|x| x.clone().into()).collect();
+        let info = get_miner_info(rt.store(), &st)?;
+
+        let (batch_return, data_activations) = activate_deals(rt, &data_activations)?;
```

Comment:
Please add similar comment to above about assumptions that mean we don't check CommD. Or maybe there's room to factor out a common `activate_deals_for_new_sector` method.